### PR TITLE
f3d: Persist configs and use raytracing

### PR DESCRIPTION
**Pull request by kazzarin**
Fixed autoupdate for the latest version: https://github.com/f3d-app/f3d/releases/tag/v2.0.0
Also fixed changes to the config file location: https://f3d.app/doc/user/CONFIGURATION_FILE.html#configuration-file

- [x] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).

### CODE REVIEW

File: bucket/f3d.json [hunk 1]
```diff
@@ -1,14 +1,20 @@
...
-            "url": "https://github.com/f3d-app/f3d/releases/download/v1.3.1/f3d-1.3.1-win64.zip",
-            "hash": "e2ffa54bbab102e55b6cd0ee9d4d2e92cac8be1093e90dade61d17bd42f2af46"
+            "url": "https://github.com/f3d-app/f3d/releases/download/v2.0.0/F3D-2.0.0-Windows-x86_64-raytracing.zip",
+            "hash": "md5:103644766af57381b93481fcbe7eff3e",
+            "extract_dir": "F3D-2.0.0-Windows-x86_64-raytracing"
         }
     },
+    "pre_install": [
+        "if (!(Test-Path \"$persist_dir\\config.json\")) {",
+        "    Copy-Item \"$dir\\share\\f3d\\configs\\config.d\\10_global.json\" \"$dir\\share\\f3d\\configs\\config.json\" -Force -ErrorAction SilentlyContinue",
```
Comment (HUMORCE) [inline bucket/f3d.json:15]: Copy `...\config.d\10_global.json` to `...\configs\config.json` is equivalent to default behavior. the whole `$dir\share\f3d\configs` should be persisted

> Windows: %APPDATA%\f3d\(config.json,.d), [install_dir]\share\f3d\configs\(config.json,.d)

Comment (kazzarin) [reply bucket/f3d.json]: Fixed. I had assumed copying one of the example configs was needed.

**Comment by kazzarin** [comment]
From the release notes:
>The Packaging of F3D is now handled by the [f3d-superbuild](https://github.com/f3d-app/f3d-superbuild) project, which let us create a nice package with correctly linked dynamic libraries instead of a single executable, which was needed to package OSPray and provide raytracing in the binary release of F3D.
>
>It also offers great flexibility and will greatly reduce any future overhead when adding dependencies, and also globally increase the quality of F3D testing.
>
>A side effect is that F3D package is now much bigger, so a package without raytracing is also provided for users looking for a smaller download and install.

Their site links to this version by default: https://f3d.app/doc/user/INSTALLATION.html

<details><summary>F3D-2.0.0-Windows-x86_64\bin</summary>

```
bin (102.66 MiB)
├─ Alembic.dll (1.41 MiB)
├─ F3DShellExtension.dll (144.00 KiB)
├─ Imath-3_1.dll (326.50 KiB)
├─ TKBO.dll (1.88 MiB)
├─ TKBRep.dll (807.50 KiB)
├─ TKBool.dll (3.24 MiB)
├─ TKG2d.dll (260.50 KiB)
├─ TKG3d.dll (797.50 KiB)
├─ TKGeomAlgo.dll (3.45 MiB)
├─ TKGeomBase.dll (3.46 MiB)
├─ TKIGES.dll (2.53 MiB)
├─ TKMath.dll (1.51 MiB)
├─ TKMesh.dll (566.50 KiB)
├─ TKPrim.dll (284.00 KiB)
├─ TKSTEP.dll (2.45 MiB)
├─ TKSTEP209.dll (395.50 KiB)
├─ TKSTEPAttr.dll (1.01 MiB)
├─ TKSTEPBase.dll (1.44 MiB)
├─ TKShHealing.dll (2.30 MiB)
├─ TKTopAlgo.dll (2.08 MiB)
├─ TKXSBase.dll (1.80 MiB)
├─ TKernel.dll (1.53 MiB)
├─ assimp-vc143-mt.dll (1.22 MiB)
├─ concrt140.dll (319.83 KiB)
├─ dbghelp.dll (1.14 MiB)
├─ draco.dll (3.80 MiB)
├─ f3d.dll (763.00 KiB)
├─ f3d.exe (457.00 KiB)
├─ msvcp140.dll (566.33 KiB)
├─ msvcp140_1.dll (34.83 KiB)
├─ msvcp140_2.dll (192.86 KiB)
├─ msvcp140_atomic_wait.dll (48.86 KiB)
├─ msvcp140_codecvt_ids.dll (30.86 KiB)
├─ tbb12.dll (303.00 KiB)
├─ vcruntime140.dll (106.83 KiB)
├─ vcruntime140_1.dll (48.36 KiB)
├─ vtkCommonColor.dll (96.00 KiB)
├─ vtkCommonComputationalGeometry.dll (164.00 KiB)
├─ vtkCommonCore.dll (5.82 MiB)
├─ vtkCommonDataModel.dll (4.20 MiB)
├─ vtkCommonExecutionModel.dll (827.00 KiB)
├─ vtkCommonMath.dll (150.50 KiB)
├─ vtkCommonMisc.dll (2.66 MiB)
├─ vtkCommonSystem.dll (160.00 KiB)
├─ vtkCommonTransforms.dll (224.00 KiB)
├─ vtkDICOMParser.dll (95.50 KiB)
├─ vtkFiltersCore.dll (7.01 MiB)
├─ vtkFiltersExtraction.dll (1.25 MiB)
├─ vtkFiltersGeneral.dll (3.58 MiB)
├─ vtkFiltersGeometry.dll (904.00 KiB)
├─ vtkFiltersHybrid.dll (758.50 KiB)
├─ vtkFiltersHyperTree.dll (210.00 KiB)
├─ vtkFiltersModeling.dll (721.00 KiB)
├─ vtkFiltersParallel.dll (710.50 KiB)
├─ vtkFiltersSources.dll (626.00 KiB)
├─ vtkFiltersStatistics.dll (782.50 KiB)
├─ vtkFiltersTexture.dll (94.50 KiB)
├─ vtkFiltersVerdict.dll (199.50 KiB)
├─ vtkIOCityGML.dll (111.00 KiB)
├─ vtkIOCore.dll (348.50 KiB)
├─ vtkIOExodus.dll (617.50 KiB)
├─ vtkIOGeometry.dll (1.60 MiB)
├─ vtkIOImage.dll (1.01 MiB)
├─ vtkIOImport.dll (302.50 KiB)
├─ vtkIOLegacy.dll (569.50 KiB)
├─ vtkIOPLY.dll (120.00 KiB)
├─ vtkIOParallel.dll (391.50 KiB)
├─ vtkIOXML.dll (796.00 KiB)
├─ vtkIOXMLParser.dll (105.50 KiB)
├─ vtkImagingColor.dll (225.50 KiB)
├─ vtkImagingCore.dll (1.57 MiB)
├─ vtkImagingGeneral.dll (623.00 KiB)
├─ vtkImagingHybrid.dll (473.00 KiB)
├─ vtkImagingMath.dll (203.00 KiB)
├─ vtkImagingSources.dll (177.00 KiB)
├─ vtkInteractionStyle.dll (212.00 KiB)
├─ vtkInteractionWidgets.dll (1.99 MiB)
├─ vtkParallelCore.dll (371.50 KiB)
├─ vtkParallelDIY.dll (716.00 KiB)
├─ vtkRenderingAnnotation.dll (751.50 KiB)
├─ vtkRenderingContext2D.dll (221.00 KiB)
├─ vtkRenderingCore.dll (1.69 MiB)
├─ vtkRenderingExternal.dll (99.00 KiB)
├─ vtkRenderingFreeType.dll (638.50 KiB)
├─ vtkRenderingHyperTreeGrid.dll (38.00 KiB)
├─ vtkRenderingOpenGL2.dll (1.90 MiB)
├─ vtkRenderingUI.dll (77.00 KiB)
├─ vtkRenderingVolume.dll (2.11 MiB)
├─ vtkRenderingVolumeOpenGL2.dll (535.50 KiB)
├─ vtkdoubleconversion.dll (64.00 KiB)
├─ vtkexodusII.dll (344.50 KiB)
├─ vtkexpat.dll (148.50 KiB)
├─ vtkfmt.dll (133.00 KiB)
├─ vtkfreetype.dll (579.00 KiB)
├─ vtkglew.dll (490.50 KiB)
├─ vtkhdf5.dll (3.30 MiB)
├─ vtkhdf5_hl.dll (120.00 KiB)
├─ vtkjpeg.dll (428.50 KiB)
├─ vtkjsoncpp.dll (208.50 KiB)
├─ vtkkissfft.dll (21.50 KiB)
├─ vtklz4.dll (113.00 KiB)
├─ vtklzma.dll (151.50 KiB)
├─ vtkmetaio.dll (522.50 KiB)
├─ vtknetcdf.dll (782.50 KiB)
├─ vtkpng.dll (194.00 KiB)
├─ vtkpugixml.dll (210.00 KiB)
├─ vtksys.dll (304.50 KiB)
├─ vtktiff.dll (386.50 KiB)
├─ vtkverdict.dll (174.00 KiB)
└─ vtkzlib.dll (87.00 KiB)
```

</details>
<details><summary>F3D-2.0.0-Windows-x86_64-raytracing\bin</summary>

```
bin (201.76 MiB)
├─ Alembic.dll (1.41 MiB)
├─ F3DShellExtension.dll (144.00 KiB)
├─ Imath-3_1.dll (326.50 KiB)
├─ OpenImageDenoise.dll (44.53 MiB)
├─ TKBO.dll (1.88 MiB)
├─ TKBRep.dll (807.50 KiB)
├─ TKBool.dll (3.24 MiB)
├─ TKG2d.dll (260.50 KiB)
├─ TKG3d.dll (797.50 KiB)
├─ TKGeomAlgo.dll (3.45 MiB)
├─ TKGeomBase.dll (3.46 MiB)
├─ TKIGES.dll (2.53 MiB)
├─ TKMath.dll (1.51 MiB)
├─ TKMesh.dll (566.50 KiB)
├─ TKPrim.dll (284.00 KiB)
├─ TKSTEP.dll (2.45 MiB)
├─ TKSTEP209.dll (395.50 KiB)
├─ TKSTEPAttr.dll (1.01 MiB)
├─ TKSTEPBase.dll (1.44 MiB)
├─ TKShHealing.dll (2.30 MiB)
├─ TKTopAlgo.dll (2.08 MiB)
├─ TKXSBase.dll (1.80 MiB)
├─ TKernel.dll (1.53 MiB)
├─ assimp-vc143-mt.dll (1.22 MiB)
├─ concrt140.dll (319.83 KiB)
├─ dbghelp.dll (1.14 MiB)
├─ draco.dll (3.80 MiB)
├─ embree3.dll (21.14 MiB)
├─ f3d.dll (763.50 KiB)
├─ f3d.exe (458.50 KiB)
├─ msvcp140.dll (566.33 KiB)
├─ msvcp140_1.dll (34.83 KiB)
├─ msvcp140_2.dll (192.86 KiB)
├─ msvcp140_atomic_wait.dll (48.86 KiB)
├─ msvcp140_codecvt_ids.dll (30.86 KiB)
├─ openvkl.dll (121.00 KiB)
├─ openvkl_module_cpu_device.dll (14.50 KiB)
├─ openvkl_module_cpu_device_16.dll (8.16 MiB)
├─ openvkl_module_cpu_device_4.dll (8.88 MiB)
├─ openvkl_module_cpu_device_8.dll (9.45 MiB)
├─ ospray.dll (188.50 KiB)
├─ ospray_module_denoiser.dll (24.50 KiB)
├─ ospray_module_ispc.dll (5.75 MiB)
├─ rkcommon.dll (83.50 KiB)
├─ tbb12.dll (303.00 KiB)
├─ tbbmalloc.dll (100.50 KiB)
├─ vcruntime140.dll (106.83 KiB)
├─ vcruntime140_1.dll (48.36 KiB)
├─ vtkCommonColor.dll (96.00 KiB)
├─ vtkCommonComputationalGeometry.dll (164.00 KiB)
├─ vtkCommonCore.dll (5.82 MiB)
├─ vtkCommonDataModel.dll (4.20 MiB)
├─ vtkCommonExecutionModel.dll (827.00 KiB)
├─ vtkCommonMath.dll (150.50 KiB)
├─ vtkCommonMisc.dll (2.66 MiB)
├─ vtkCommonSystem.dll (160.00 KiB)
├─ vtkCommonTransforms.dll (224.00 KiB)
├─ vtkDICOMParser.dll (95.50 KiB)
├─ vtkDomainsChemistry.dll (210.50 KiB)
├─ vtkFiltersCore.dll (7.01 MiB)
├─ vtkFiltersExtraction.dll (1.25 MiB)
├─ vtkFiltersGeneral.dll (3.58 MiB)
├─ vtkFiltersGeometry.dll (904.00 KiB)
├─ vtkFiltersHybrid.dll (758.50 KiB)
├─ vtkFiltersHyperTree.dll (210.00 KiB)
├─ vtkFiltersModeling.dll (721.00 KiB)
├─ vtkFiltersParallel.dll (710.50 KiB)
├─ vtkFiltersSources.dll (626.00 KiB)
├─ vtkFiltersStatistics.dll (782.50 KiB)
├─ vtkFiltersTexture.dll (94.50 KiB)
├─ vtkFiltersVerdict.dll (199.50 KiB)
├─ vtkIOCityGML.dll (111.00 KiB)
├─ vtkIOCore.dll (348.50 KiB)
├─ vtkIOExodus.dll (617.50 KiB)
├─ vtkIOGeometry.dll (1.60 MiB)
├─ vtkIOImage.dll (1.01 MiB)
├─ vtkIOImport.dll (302.50 KiB)
├─ vtkIOLegacy.dll (569.50 KiB)
├─ vtkIOPLY.dll (120.00 KiB)
├─ vtkIOParallel.dll (391.50 KiB)
├─ vtkIOXML.dll (796.00 KiB)
├─ vtkIOXMLParser.dll (105.50 KiB)
├─ vtkImagingColor.dll (225.50 KiB)
├─ vtkImagingCore.dll (1.57 MiB)
├─ vtkImagingGeneral.dll (623.00 KiB)
├─ vtkImagingHybrid.dll (473.00 KiB)
├─ vtkImagingMath.dll (203.00 KiB)
├─ vtkImagingSources.dll (177.00 KiB)
├─ vtkInteractionStyle.dll (212.00 KiB)
├─ vtkInteractionWidgets.dll (1.99 MiB)
├─ vtkParallelCore.dll (371.50 KiB)
├─ vtkParallelDIY.dll (716.00 KiB)
├─ vtkRenderingAnnotation.dll (751.50 KiB)
├─ vtkRenderingContext2D.dll (221.00 KiB)
├─ vtkRenderingCore.dll (1.69 MiB)
├─ vtkRenderingExternal.dll (99.00 KiB)
├─ vtkRenderingFreeType.dll (638.50 KiB)
├─ vtkRenderingHyperTreeGrid.dll (38.00 KiB)
├─ vtkRenderingOpenGL2.dll (1.90 MiB)
├─ vtkRenderingRayTracing.dll (421.00 KiB)
├─ vtkRenderingSceneGraph.dll (69.00 KiB)
├─ vtkRenderingUI.dll (77.00 KiB)
├─ vtkRenderingVolume.dll (2.11 MiB)
├─ vtkRenderingVolumeOpenGL2.dll (535.50 KiB)
├─ vtkdoubleconversion.dll (64.00 KiB)
├─ vtkexodusII.dll (344.50 KiB)
├─ vtkexpat.dll (148.50 KiB)
├─ vtkfmt.dll (133.00 KiB)
├─ vtkfreetype.dll (579.00 KiB)
├─ vtkglew.dll (490.50 KiB)
├─ vtkhdf5.dll (3.30 MiB)
├─ vtkhdf5_hl.dll (120.00 KiB)
├─ vtkjpeg.dll (428.50 KiB)
├─ vtkjsoncpp.dll (208.50 KiB)
├─ vtkkissfft.dll (21.50 KiB)
├─ vtklz4.dll (113.00 KiB)
├─ vtklzma.dll (151.50 KiB)
├─ vtkmetaio.dll (522.50 KiB)
├─ vtknetcdf.dll (782.50 KiB)
├─ vtkpng.dll (194.00 KiB)
├─ vtkpugixml.dll (210.00 KiB)
├─ vtksys.dll (304.50 KiB)
├─ vtktiff.dll (386.50 KiB)
├─ vtkverdict.dll (174.00 KiB)
└─ vtkzlib.dll (87.00 KiB)
```

</details>

The additional dlls included in the raytracing version are:
- OpenImageDenoise.dll
- embree3.dll
- openvkl.dll
- openvkl_module_cpu_device.dll
- openvkl_module_cpu_device_16.dll
- openvkl_module_cpu_device_4.dll
- openvkl_module_cpu_device_8.dll
- ospray.dll
- ospray_module_denoiser.dll
- ospray_module_ispc.dll
- rkcommon.dll
- tbbmalloc.dll
- vtkDomainsChemistry.dll
- vtkRenderingRayTracing.dll
- vtkRenderingSceneGraph.dll

tl;dr the non-raytracing version only exists to save space

**Review by HUMORCE — APPROVED** [review]
